comeet/Data/Authenticator/AuthenticatorADALImplementer.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AuthenticatorADALImplementer.swift
+//  comeet
+//
+//  Created by Ricardo Contreras on 3/6/17.
+//  Copyright © 2017 teamawesome. All rights reserved.
+//
+
+import Foundation
+import ADAL
+
+class AuthenticatorADALImplementer : AuthenticatorProtocol
+{
+    private struct Constants {
+        static let authority = "https://login.microsoftonline.com/common"
+        static let resource = "https://outlook.office365.com"
+        static let clientId = "a64d56ea-5675-4ccf-82e9-5757620e1d26"
```

Comment:
@kb000 It is the App registration on Azure. I'll send a change to avoid using it directly.